packages/schema/docs/build/schema.md

```diff
@@ -145,6 +148,91 @@ Key | Required | Type | Description
 
 -----
 
+## /AuthFieldSchema
+
+Field schema specialized for authentication fields.
+
+#### Details
+
+* **Type** - `object`
+* [**Source Code**](https://github.com/zapier/zapier-platform/blob/zapier-platform-schema@16.2.0/packages/schema/lib/schemas/AuthFieldSchema.js)
+
+#### Properties
+
+Key | Required | Type | Description
+--- | -------- | ---- | -----------
+`key` | **yes** | `string` | A unique machine readable key for this value (IE: "fname").
+`label` | no | `string` | A human readable label for this value (IE: "First Name").
+`helpText` | no | `string` | A human readable description of this value (IE: "The first part of a full name."). You can use Markdown.
+`type` | no | `string` in (`'string'`, `'text'`, `'integer'`, `'number'`, `'boolean'`, `'datetime'`, `'file'`, `'password'`, `'copy'`, `'code'`) | The type of this value. Use `string` for basic text input, `text` for a large, `<textarea>` style box, and `code` for a `<textarea>` with a fixed-width font. Field type of `file` will accept either a file object or a string. If a URL is provided in the string, Zapier will automatically make a GET for that file. Otherwise, a .txt file will be generated.
```

Comment:
This should be limited to what actually works.

Just from [rendering](https://cdn.zappy.app/07033c49fc406de8261e71fbf8108335.gif) it seems `code` doesn't, but I also doubt `file` would actually process well.

Reply:
That's really neat! How did you get it to render? I tried running the monolith and editor locally then create an app using CLI locally, but I wasn't able to see it via the editor, any tips would be awesome!

Reply:
Yes, that's what I did as well.

Reply:
Argh, Natay just called my attention to this suggestion being for AuthFieldSchema and not necessarily for InputFieldSchema. I agree then. Thanks @FokkeZB

Reply:
Thanks both for the guidance! I was able to [render](https://cdn.zappy.app/dab1dd0f286c5e998a7dd650aa6ccfad.gif) and keep only what works.